spotinst/ocean_aks_np_vm_sizes/fields_spotinst_ocean_aks_np_vm_sizes.go

```diff
@@ -235,6 +337,24 @@ func flattenFilters(filters *azure_np.Filters) []interface{} {
 		if filters.MaxMemoryGiB != nil {
 			result[string(MaxMemoryGiB)] = spotinst.Float64Value(filters.MaxMemoryGiB)
 		}
+		if filters.MinGpu != nil {
```

Comment:
please add result[string(MaxMemoryGiB)] = value this for number data types attributes

Reply:
added for mingpu maxgpu minnics and mindata